Frontend/src/pages/register/components/CompetingStep.jsx

```diff
@@ -105,7 +105,7 @@ export default function CompetingStep({ nextStep }) {
   const commentIsValid =
     comment.trim() || !competitionInfo.force_comment_in_registration
   // TODO: get max events can register for
-  const maxEvents = Infinity
+  const maxEvents = competitionInfo.events_per_registration_limit
```

Comment:
This might be necessary?
```suggestion
  const maxEvents = competitionInfo.events_per_registration_limit ?? Infinity
```

Reply:
Great catch, thanks!